### PR TITLE
LANS Clean Up #6 - User Class

### PR DESCRIPTION
**Pull request by MikeyGMT**
LANS Clean Up #6 - User Class - more to do within conflicting defines
lan_user.php

### CODE REVIEW

**Comment by MikeyGMT** [comment]
Testing 

image

image

image

File: e107_admin/userclass2.php [hunk 1]
```diff
@@ -391,7 +391,7 @@ public function initialPage()
 				    <td>".$frm->select('init_class_stage', $initClassStages, $sel_stage)."<span class='field-help'>".UCSLAN_46."</span>
 
 				    </td></tr></table>
-				    <div class='buttons-bar'>".	$frm->admin_button('set_initial_classes','no-value','create',LAN_UPDATE)."</div>";
+				    <div class='buttons-bar' center>".	$frm->admin_button('set_initial_classes','no-value','create',LAN_UPDATE)."</div>";
```
Comment (CaMer0n) [inline e107_admin/userclass2.php]: I believe the 'center' should be inside the single-quote?

File: e107_admin/userclass2.php [hunk 2]
```diff
@@ -718,16 +718,16 @@ function checkAllowed($classID, $redirect = true, $fullEdit = FALSE)
 		//		unset($pref['frontpage'][$class_id]);		// (Should work with both 0.7 and 0.8 front page methods)
 		//		save_prefs();
 			}
-			$emessage->add(UCSLAN_3, E_MESSAGE_SUCCESS);
+			$emessage->add(UCSL/AN_3, E_MESSAGE_SUCCESS);
```
Comment (CaMer0n) [inline e107_admin/userclass2.php]: What's happening here?

File: e107_admin/userclass2.php (hunk 1 above)
Comment (MikeyGMT) [inline e107_admin/userclass2.php]: Oops. I was thinking that when I added it. Silly me. Correct it tomorrow.

File: e107_admin/userclass2.php (hunk 2 above)
Comment (MikeyGMT) [inline e107_admin/userclass2.php]: Stops old lans in commented blocks from falsely appearing as missing must be renabled in deprecated LANs checker. My bad?

Comment (CaMer0n) [inline e107_admin/userclass2.php]: Well UCSLAN_3 is "Class deleted." right? So, replace with LAN_DELETED should be enough I guess.

File: e107_admin/userclass2.php (hunk 1 above)
Comment (MikeyGMT) [inline e107_admin/userclass2.php]: It's this 
`<div class='buttons-bar center'>`
not this right?   
`<div class='buttons-bar' 'center'>`

Oops No 3

**Comment by CaMer0n** [comment]
looks good @MikeyGMT !!

**Pull request merged**